tgui/packages/tgui/interfaces/OverwatchConsole.js

```diff
@@ -0,0 +1,860 @@
+                        fluid
+                        content={marine_data['name']}
+                        onClick={() =>
+                          act('use_cam', {
+                            cam_target: marine_data['ref'],
+                          })}
+                      />
+                    </Table.Cell>
+                    <Table.Cell>
+                      {marine_data['role']
+                        + (marine_data['act_sl'] === true
+                          ? marine_data['act_sl']
+                          : '')
+                        + (marine_data['fteam']
+                          ? ' (' + marine_data['fteam'] + ')'
+                          : '')}
+                    </Table.Cell>
+                    <Table.Cell>
+                      <ColorBox
+                        color={
+                          marine_data['mob_state'] === 'Conscious'
+                            ? 'green'
+                            : 'red'
+                        }
+                      />
+                      {' ' + marine_data['mob_state']}{' '}
+                      {!marine_data['helmet'] ? ' (no helmet)' : null}
+                      {marine_data.SSD ? ' (SSD)' : null}
+                    </Table.Cell>
+                    <Table.Cell>
+                      {marine_data['area_name']
+                        ? marine_data['area_name']
+                        : 'N/A'}
+                    </Table.Cell>
+                    <Table.Cell>
+                      {marine_data['dist'] ? marine_data['dist'] : 'N/A'}
+                      {marine_data['dir'] ? " (" + marine_data['dir'] + ")" : " (N/A)"}
+                    </Table.Cell>
+                    <Table.Cell>
+                      <Button
+                        content={
+                          marine_data.manually_filtered ? 'SHOW' : 'HIDE'
+                        }
+                        selected={marine_data.manually_filtered}
+                        textColor={
+                          !marine_data.manually_filtered ? '#161613' : '#fffff'
+                        }
+                        onClick={() =>
+                          act('filter_marine', {
+                            squaddie: marine_data.ref,
+                          })}
+                      />
+                    </Table.Cell>
+                  </Table.Row>
+                ))}
+            </Table>
+          </Stack.Item>
+        </Section>
+        <Stack.Item>
+          <Table>
+            <Table.Row>
+              <Table.Cell>Squad Leader</Table.Cell>
+              <Table.Cell>RTO</Table.Cell>
+              <Table.Cell>Specialist</Table.Cell>
+              <Table.Cell>Smartgunner</Table.Cell>
+              <Table.Cell>Medic</Table.Cell>
+              <Table.Cell>Combat Technician</Table.Cell>
+              <Table.Cell>Alive</Table.Cell>
+              <Table.Cell>Deployed</Table.Cell>
+            </Table.Row>
+            <Table.Row>
```

Comment:
Why not have a single cell, which spans all rows and contains a divider? Alternatively, style the above row to have a border on the bottom